projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/graphql/model/GraphQLResponse.java

```diff
@@ -3,14 +3,18 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package com.vmware.taurus.service.model;
+package com.vmware.taurus.service.graphql.model;
 
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
 
 import java.util.List;
 
 @Data
-public class DataJobPage {
+@Builder
+@AllArgsConstructor
+public class GraphQLResponse {
```

Comment:
Why rename it? It still returns Data Jobs - only information about their executions.

Reply:
Yes, you have the point.